core/new-gui/package.json

```diff
@@ -77,6 +79,7 @@
     "ngx-bootstrap": "~5.6.2",
     "ngx-color-picker": "^12.0.1",
     "ngx-json-viewer": "~3.0.2",
+    "ngx-markdown": "~13.1.0",
```

Comment:
hmm, by specific I mean use "13.1.0" instead of `~13.1.0` or `^13.1.0`. Sometimes minor updates can fail the build for some reason... so we want to avoid any implicit updates.

Reply:
Sounds good, updated to fix this